BlitzBoard/Features/HomeScreenTableCell/TableViewCellViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TableViewCellViewController.swift
+//  BlitzBoard
+//
+//  Created by Kiasha Rangasamy on 2024/04/17.
+//
+
+import UIKit
+
+class StandingTableViewCell: UITableViewCell {
```

Comment:
These names don't make sense.. This is a homescreen feature?
Plese name everything accordingly in the code.
HomeScreenViewController, HomeScreenViewModel etc